server_manager/www/shadowbox_server.ts

```diff
@@ -260,8 +277,39 @@ export class ShadowboxServer implements server.Server {
     return await this.api.request<ServerConfigJson>('server');
   }
 
+  private async getSupportedExperimentalUniversalMetricsEndpoint(): Promise<boolean> {
+    if (this._supportedExperimentalUniversalMetricsEndpointCache !== null) {
+      return this._supportedExperimentalUniversalMetricsEndpointCache;
+    }
+
+    let result = false;
+
+    if (!this.api) return result;
+
+    try {
+      await this.api.request<server.ServerMetricsJson>(
+        'experimental/server/metrics?since=30d',
+        'HEAD'
+      );
+      result = true;
+    } catch (error) {
+      // endpoint is not defined, keep set to false
+      if (error.response?.status !== 404) {
```

Comment:
If we don't have a status (e.g. connection error), we should probably return false and keep the cache null to try again next time